index.html

```diff
@@ -612,10 +633,12 @@ <h4 class="subtitle is-6">Our way to success</h4>
           </div>
         </div>
       </div>
-      <button class="modal-close is-large"></button>
+      <!-- WARNING: better to use button tag, added for HW -->
+      <div role="button" class="modal-close is-large"></div>
     </div>
 
-    <div id="modal-image1" class="modal modal-fx-3dSlit">
+    <!-- TODO: aria-modal should be set to true when modal is opened -->
+    <div id="modal-image1" class="modal modal-fx-3dSlit" role="dialog" aria-modal="false">
```

Comment:
Also as part of improvement you can aria-describedby and aria-labelledby attr
Please check example here https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-modal